plugins/emoticons.md

```diff
@@ -78,9 +78,35 @@ tinymce.init({
 });
 ```
 
+### `emoticons_database`
+
+This option provides the ability to specify which built-in emoji database to use when rendering emojis in the editor. The following built-in emoji databases are available:
+* `emojis` - This database uses unicode characters to represent emoji in the editor content.
+* `emojiimages` - This database uses images provided by Twitter Emoji (twemoji) to represent emoji in the editor content.
```

Comment:
```suggestion
* `emojiimages` - This database uses images provided by the Twitter Emoji (twemoji) library to represent emoji in the editor content.
```

Reply:
Hmm... it's not really the library we're using. Maybe `graphics` instead?

Reply:
hmmm project?

Reply:
That works :+1: